app/frontend/src/components/forms/manage/ManageVersions.vue

```diff
@@ -57,18 +57,24 @@
       <!-- Status  -->
       <template #[`item.status`]="{ item }">
         <v-switch
+          :class="{ 'dir-rtl': isRTL }"
```

Comment:
Hmm, something broke the switch when viewing in RTL. It looks OK in dev, but in this PR the round circle isn't in the right place:

<img width="88" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/6fdd9e3e-0908-4af7-b517-be9deb34b1ae">

Reply:
This is not from us. It is from Vuetify. I will raise the issue with them but I have tried to see what can be done on our end but nothing is working

Reply:
I've unresolved this conversation as this can be resolved by removing the RTL on the switch itself. 
![image](https://github.com/bcgov/common-hosted-form-service/assets/101672465/23e5afd5-5058-4a4b-a586-78ccfdd7eff7)
![image](https://github.com/bcgov/common-hosted-form-service/assets/101672465/5de28d2d-bb92-48c2-a3cd-9ddb78f80e95)